models/metadata.go

```diff
@@ -26,33 +28,38 @@ type Metadata struct {
 	Classifications string               `json:"classifications,omitempty"`
 	Source          string               `json:"source,omitempty"`
 	IsBasedOn       *IsBasedOn           `json:"is_based_on,omitempty"`
+	Type            string               `json:"type,omitempty"`
+	Version         int                  `json:"version,omitempty"`
```

Comment:
I think topics needs adding to this list as it's in the tier 0 model but is not returned when I add it in manually to a database record

Reply:
This will be addressed in https://jira.ons.gov.uk/browse/DIS-2818 as discussed